src/issue/screens/pull-merge.screen.js

```diff
@@ -68,9 +68,22 @@ class PullMerge extends Component {
     };
   }
 
+  mergeMethodMessages = ['Create a merge commit', 'Squash and merge'];
+
+  showActionSheet = () => {
+    this.ActionSheet.show();
+  };
+
+  handlePress = (i) => {
+    if (i !== this.mergeMethodMessages.length) {
```

Comment:
What does `i` stand out for? I think it doesn't hurt to add a more descriptive name in here.